cgmanifest.json

```diff
@@ -24514,7 +24514,7 @@
         "other": {
           "name": "python-sphinxcontrib-apidoc",
           "version": "0.2.1",
```

Comment:
Please change the version to upgraded one(0.3.0).

Reply:
Updated the cgmanifest.json file with upgraded version(0.3.0).